02_getting_started.asciidoc

```diff
@@ -188,44 +188,44 @@ Both Bitcoin addresses and QR codes contain error detection information that pre
 
-Similarly, if she billed a client for a service offered over the Internet, Alice could send an email or instant message with the Bitcoin Address or the QR code (or both) to her client and they could scan or paste the information into a Bitcoin wallet to pay her.
+Similarly, if she billed a client for a service offered over the Internet, Alice could send an email or instant message with the Bitcoin Address or the QR code to her client, and they could paste or scan the information into a Bitcoin wallet to pay her.
 
-Alice could even print the QR code and affix it to a sign and display it to receive tips. A singer who receives bitcoin tips has a QR code printed and affixed to their guitar while performing!
+Alice could even print the QR code and affix it to a sign and display it to receive tips. For example, she could have a QR code affixed to her guitar and receive tips while performing!
 
 Finally, if Alice bought bitcoin from a crypto-currency exchange, she could (and should!) then "withdraw" the bitcoin by pasting her Bitcoin Address into the exchange website. The exchange will then send the bitcoin to her address directly.
 
 === From Bitcoin to Lightning Network
 
-Alice's bitcoin is now controlled by her Eclair wallet and has been recorded on the Bitcoin Blockchain. At this point, Alice's bitcoin is "on-chain", meaning that she has completed a regular Bitcoin transaction, broadcast to the entire Bitcoin network, verified by all Bitcoin nodes and "mined" (recorded) onto the Bitcoin blockchain.
+Alice's bitcoin is now controlled by her Eclair wallet and has been recorded on the Bitcoin Blockchain. At this point, Alice's bitcoin is "on-chain," meaning that the transaction has been broadcast to the entire Bitcoin network, verified by all Bitcoin nodes, and "mined" (recorded) onto the Bitcoin blockchain.
 
-So far, the Eclair Mobile wallet has behaved only as a Bitcoin wallet and Alice hasn't used the Lightning Network features of Eclair. As is the case with many LN wallets, Eclair bridges Bitcoin and the Lightning Network by acting as both a Bitcoin wallet and a Lightning wallet.
+So far, the Eclair Mobile wallet has behaved only as a Bitcoin wallet, and Alice hasn't used the Lightning Network features of Eclair. As is the case with many LN wallets, Eclair bridges Bitcoin and the Lightning Network by acting as both a Bitcoin wallet and a Lightning wallet.
 
-Now, Alice is ready to start using the Lightning Network by taking her bitcoin "off-chain", so she can take advantage of the fast, cheap and private payments that LN offers.
+Now, Alice is ready to start using the Lightning Network by taking her bitcoin "off-chain," so she can take advantage of the fast, cheap, and private payments that the Lightning Network offers.
```

Comment:
we try to User more LN instead of Lightning Network (so the other use of Lightning Network should probably have been abbriviated.) I will accept this PR thought since the rest looks good to me and the Lightning Network LN thing will probably be in a final review anyway.